src/Discord/Builders/MessageBuilder.php

```diff
@@ -632,6 +640,16 @@ public function setFlags(int $flags): self
         return $this;
     }
 
+    /**
+     * Get the current flags of the message.
+     *
+     * @return ?int
+     */
+    public function getFlags(): ?int
+    {
+        return $this->flags ?? null;
```

Comment:
Would it make more sense to return 0 instead of null?

Reply:
this sounds good

Reply:
Done and updated relevant parts